src/cnc/flavors/aws/lambda-lite/1/provision/base.tf.j2

```diff
@@ -0,0 +1,37 @@
+{% block state %}
+terraform {
+  backend "local" {
+    path = "{{ config_renderer.working_dir }}/.terraform/{{ config_renderer.tf_state_namespace }}.tfstate"
+  }
+}
+{% endblock %}
+
+{% if not output_only %}
+
+{% block essentials %}
+{% include "partials/variables/environments.tf.j2" %}
+{% include "partials/collection/providers.tf.j2" %}
```

Comment:
We do need a block called `providers` here with just the providers partial. The reason we do this is so that just the provider configuration can be overwritten.

Reply:
Ajustments Commit: https://github.com/tmaior/cnc-fork/commit/9ed6e46909e2e43c43eee39bb138ed6d2df70b18